api.md

```diff
@@ -170,6 +171,114 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
+| `node_name` | Name of the node. |
+
+##### Parameter Description
+
+* `node_name` - Name to use for the node locally.
+
+#### Optional Arguments [](add_data_node-optional-arguments)
+
+| Name                 | Description                                           |
+|----------------------|-------------------------------------------------------|
+| `host`               | Host name for the remote data node.                   |
+| `port`               | Port to use on the remote data node.                  |
+| `database`           | Database name where the remote hypertable is created. |
+| `local_user`         | User name on the local node.                          |
+| `remote_user`        | User name on the remote data node.                    |
+| `password`           | Password on the remote data node.                     |
+| `if_not_exists`      | Do not fail if the hyper table already exists         |
+| `bootstrap_database` | Database to bootstrap.                                |
+| `bootstrap_user`     | User to used for bootstrapping.                       |
+| `bootstrap_password` | Password for user use when bootstrapping.             |
+
+##### Parameter Description
+
+* `host` - the name of the host to use for the remote data node. This
+  defaults to `'localhost'`.
+
+* `post` - Port to use for the remote data node. This defaults to the
+  default PostgreSQL port.
+
+* `database` - Name of the database to use on the remote data
+  node. Default is to use the current database in use when calling the
+  function.
+
+* `local_user` - Name of the local user. Default is to use the current
+  user.
+
+* `remote_user` - Name of the user on the remote data node. Default is
+  to use the same user as `local_user`.
+
+* `password` - Password to use on the remote data node. Defaults is to
+  not use a password.
+
+* `bootstrap_database` - The database to use when bootstrapping the
+  server as described above. The default is to use `'postgres'`.
+
+* `bootstrap_user` - User to use when bootstrapping the server as
+  described above. The default is to use the saver provided as
+  `remote_user` (or the default).
+
+* `bootstrap_password` - Password to use when bootstrapping the server
+  as described above. The default is to use the password as provided
+  as `password`.
+
+#### Returns [](add_data_node-returns)
+
+| Column          | Description                                   |
+|-----------------|-----------------------------------------------|
+| `node_name`     | Name of the data node                         |
+| `host`          | Host name for the remote data node            |
+| `port`          | Port for the remote data node                 |
+| `database`      | Name of database used on the remote data node |
+| `username`      | Local username                                |
+| `node_username` | Remote username                               |
+| `created`       |                                               |
```

Comment:
What is type of created?  It this a boolean or time?

Reply:
`created` is  a boolean that indicates whether the remote database was created or not (needed because of `if_not_exists`parameter).

Reply:
Fixed.